templates/multistage-buildconfig.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    description: Template to produce an imagestream and buildconfig for a Jlinked image
+  name: jlinked-image-template
+objects:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: jlinked-image
+  spec:
+    lookupPolicy:
+      local: false
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: ubi9-jlinked-image-buildconfig
+  spec:
+    source:
+      dockerfile: |
+        #Stage-1: ubi9-jlinked-image is builder image + application + jlinked JVM
+        # install additional system dependencies (for ubi-micro) to /mnt/jrootfs
+        FROM -
```

Comment:
until the `dockerStategy.from.name` below is a real image stream, I'd put the test image URI  here: `FROM quay.io/repository/jdowland/jlink:ubi9-jlinked-image`